index.css

```diff
@@ -112,3 +112,235 @@ ul {
   position: absolute;
   left: 130px;
 }
+
+.work {
+  display: grid;
+  grid-template-columns: none;
+  grid-template-rows: none;
+  justify-content: space-evenly;
+  padding-right: 15px;
+}
+
+#button1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 90px;
+  background-color: #7d73fc;
+  height: 35px;
+  text-align: center;
+  font-size: 12px;
+  font-family: 'Roboto', sans-serif;
+  color: #edecff;
+}
+
+#button1:hover {
+  cursor: pointer;
+  background-color: purple;
+}
+
+.lang {
+  border-right: 1px solid gray;
+  border-width: 1px;
+  padding-right: 15px;
+}
+
+.lang:last-child {
+  border: none;
+}
+
+#list1 {
+  display: flex;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  color: #fff;
+  padding-right: 15px;
+}
+
+#sub-heading1 {
+  display: flex;
+  align-items: center;
+  font-family: 'Roboto', sans-serif;
+  font-size: 32px;
+  color: #fff;
+}
+
+#content1 {
+  display: flex;
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 20px;
+  color: #dcdce8;
+}
+
+.img-container1 {
+  background-image: url(resources/Ellipse\ 19@2x.png);
+  background-repeat: no-repeat;
+  background-position: left top;
+  background-size: 100% 100%;
+}
+
+#heading {
+  display: flex;
+  justify-content: center;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  color: #9490ff;
+  border-bottom: 2px solid #646369;
+  flex-wrap: wrap;
+}
+
+#button2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 90px;
+  background-color: #7d73fc;
+  height: 35px;
+  text-align: center;
+  font-size: 12px;
+  font-family: 'Roboto', sans-serif;
+  color: #edecff;
+}
+
+#button2:hover {
+  background-color: purple;
+  cursor: pointer;
+}
+
+#list2 {
+  display: flex;
+  align-items: cemter;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  color: #fff;
+}
+
+#sub-heading2 {
```

Comment:
Please use the good font-weight here it is `font-weight: 500;`

Reply:
:heavy_check_mark: Good job at handling with this issue. :+1: